uportal-db/includes/00-uportal-user-and-db.sql

```diff
@@ -0,0 +1,3 @@
+CREATE DATABASE uportal;
+CREATE USER uportal WITH SUPERUSER PASSWORD 'password';
+ALTER USER postgres WITH SUPERUSER PASSWORD 'password';
```

Comment:
I was looking back through this and I think at least here the best option is to go with the postgres container's options for using `POSTGRES_USER` and `POSTGRES_PASSWORD`. That lets us use some more native k8s concepts, and keep things flexible for end users. Good call. :grin: